sql/core/src/test/scala/org/apache/spark/sql/jdbc/JDBCWriteSuite.scala

```diff
@@ -574,6 +576,41 @@ class JDBCWriteSuite extends SharedSparkSession with BeforeAndAfter {
     }
   }
 
+  test("SPARK-32013: option preActions/postActions, run SQL before writing data.") {
```

Comment:
nit: Basically, we don't need the prefix, e.g., SPARK-32013, when adding new features.

Reply:
Removed the prefix.